src/pages/docs/lexicon/image_card.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Image Card
+description: ""
+layout: "lexiconMigrationGuide"
+weight: 101
+
+clayComponentLink: "https://claycss.com/docs/components/cards.html"
+lexiconPattern: "Image Card"
+lexiconPatternLink: "https://lexicondesign.io/docs/patterns/cards.html"
+---
+
+<article id="before-after">
+
+### How does it look like now (7.0)
+
+<img class="img img-thumbnail" src="/images/lexiconMigration/image_card_old.png">
+
+### Changes for Lexicon 2.0 (7.1)
+
+<img class="img img-thumbnail" src="/images/lexiconMigration/image_card_new.png">
+
+Change | Reason
+--- | ---
+① The **icons spritemap** has changed | Fit with the new Lexicon icons
+② The **link style** now is black, blue on hover. Also the entire card has hover state | Fit with new links styles and better visual feedback
+③ The card no supports labels
```

Comment:
This breaks the table. Also previously the card accepted labels, but not through API but through content.

Reply:
There are more changes like the ratio of the card, the distribution of the details...